### DISCUSSION TITLE
How to import my own youtube playlist

### DISCUSSION THREAD

**Discussion by legouju** [snuffyDev/Beatbump#141] [General]
I was wondering how would I import my youtube playlist.

Do I have to export it as a json first somehow or is there an easier way.
If I have to export it as a json any recommendations apps I should use for that?

Thanks

**Comment by snuffyDev** [comment]
At the moment, there’s no way to import from YTM, aside from making your playlist public, and then going to it on Beatbump and hitting “Add to Playlist” and creating one that way.

This does give me an idea for finding some sort of solution though 🤔

**Comment by JBuildr** [comment]
> At the moment, there’s no way to import from YTM, aside from making your playlist public, and then going to it on Beatbump and hitting “Add to Playlist” and creating one that way.
> 
> This does give me an idea for finding some sort of solution though 🤔

And do u found a solution?

**Reply by Revdep** [reply]
Hello, any update on this? 
I make my playlist public but i can't find a way to import on beatbump 🤔
